packages/api-clients/open-api/apiGatewayApi.yml

```diff
@@ -2665,7 +2665,7 @@ components:
         code:
           type: string
           description: source identifier
-          pattern: '[a-z0-9 \-]{,48}'
+          pattern: '^[a-zA-Z0-9\-_]+$'
```

Comment:
Making this pattern match the one of the attribute seed, otherwise it fails parsing when returning an object with data that was accepted on creation

<img width="448" alt="image" src="https://github.com/user-attachments/assets/347343d0-3338-431c-aef0-42ea30faa347">